pkg/pkg.ml

```diff
@@ -0,0 +1,7 @@
+#use "topfind"
+#require "topkg-jbuilder"
```

Comment:
Don't I need to use the manual one because jbuilder will not be able to pick up the project name automatically because of mirage-http?